src/main/java/htsjdk/samtools/cram/encoding/reader/RefSeqIdReader.java

```diff
@@ -1,242 +0,0 @@
-/*******************************************************************************
```

Comment:
This was mostly copy/paste from CramRecordReader, with a few additions to collect spans